### ISSUE TITLE
tell wikipedia: E influences SES and Monte

### ISSUE THREAD

**Issue by dckc** [dckc/awesome-ocap#11]
revise **Influences** section of infobox on https://en.wikipedia.org/wiki/E_(programming_language) to add SES and Monte

Bonus points to anyone who beats me to it.

cc @erights @agoric @MostAwesomeDude

p.s. Out of concern for WP:COI, I haven't made edits around capability security in wikipedia to date, but I think the conflict is manageable.

**Comment by erights** [comment]
E also influenced JavaScript

**Comment by dckc** [comment]
> What conflict?

Right... when I look at it closely, I don't have much of a conflict. But...

I do work on Monte, so citing it borders on [WP:SELFCITE](https://en.wikipedia.org/wiki/Wikipedia:Conflict_of_interest#Citing_yourself). That sort of thing.
